.dockerignore

```diff
@@ -8,3 +8,6 @@
 /keys
 package-lock.json
 node_modules
+cvat-ui
+cvat-core/node_modules
```

Comment:
Doesn't the line 10 work?

Reply:
It seems it doesn't. Better to have '**/node_modules' pattern here.